### PR TITLE
Centrifuge App: Asset detail page changes

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

Add loan document info to asset detail page and add pricing form to sidebar. Bunch of data still mocked until we can integrate the node

### Approvals

- [ ] Dev

### Screenshots

image

image

### Impact

Centrifuge App

### CODE REVIEW

File: centrifuge-app/src/pages/Loan/index.tsx [hunk 1]
```diff
@@ -37,15 +50,23 @@ export const LoanPage: React.FC = () => {
...
   const { pid, aid } = useParams<{ pid: string; aid: string }>()
   const loan = useLoan(pid, aid)
+  const pool = usePool(pid)
   const address = useAddress()
   const permissions = usePermissions(address)
   const canBorrow = useCanBorrow(pid, aid)
+  const canPrice = permissions?.pools[pid]?.roles.includes('PricingAdmin')
+
+  if (loan && pool && loan?.status === 'Created' && canPrice) {
+    return <PricingForm loan={loan} pool={pool} />
```
Comment (sophialittlejohn) [inline centrifuge-app/src/pages/Loan/index.tsx:60]: If I recall properly, we'll need a different variation of the PricingForm for altair and for centrifuge, maybe you can pass the `config.network` as a prop 🤔

Comment (onnovisser) [reply centrifuge-app/src/pages/Loan/index.tsx:60]: Ah, yeah Altair might only do the bullet loans. I'll just have the PricingForm itself handle that

Comment (onnovisser) [reply centrifuge-app/src/pages/Loan/index.tsx:60]: removed the loan type dropdown on altair

**Review by sophialittlejohn — APPROVED** [review]
Looks good for now 👍  will be nice to integrate the node

**Pull request merged**